test/unit/lib/postcode_sanitizer_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class PostcodeSanitizerTest < ActiveSupport::TestCase
+
+  context "postcodes come through to imminence" do
+
+    should "strip trailing spaces from entered postcodes" do
+      assert_equal "WC2B 6NH", PostcodeSanitizer.sanitize("WC2B 6NH ")
+    end
+
+    should "strip non-alphanumerics except underscores from entered postcodes" do
+      # Allowing underscores is a feature of the `\w` regex.
```

Comment:
This doesn't seem like a good reason to keep underscores...  Could you not just use a character class in the regex (eg `/[^a-z0-9 ]/i`